awx/sso/conf.py

```diff
@@ -1603,6 +1603,50 @@ def get_saml_entity_id():
     ],
 )
 
+register(
+    'LOCAL_PASSWORD_MIN_LENGTH',
+    field_class=fields.IntegerField,
+    min_value=0,
```

Comment:
does the django validation have a minimum? would it make sense to restrict the min to whatever that value is?

Reply:
Technically it can't be blank but you can set a password to `a` (so min 1) but I wanted `0` to mean "check disabled".